lib/src/date_time.dart

```diff
@@ -50,6 +50,9 @@ class TZDateTime implements DateTime {
   /// Represents the same moment as this [TZDateTime].
   final DateTime _native;
 
+  /// Returns a native DateTime object representing this same point in time.
+  DateTime get native => _native;
```

Comment:
You can just make the field above public, instead of having a field and getter.

https://dart.dev/effective-dart/usage#prefer-using-a-final-field-to-make-a-read-only-property